src/modlist.h

```diff
@@ -55,6 +57,33 @@ class ModList : public QAbstractItemModel
 
 public:
 
+  enum ModListRole {
+
+    // data(GroupingRole) contains the "group" role - This is used by the
+    // category and Nexus ID grouping proxy (but not the ByPriority proxy)
+    GroupingRole = Qt::UserRole,
+
+    IndexRole = Qt::UserRole + 1,
+
+    // data(AggrRole) contains aggregation information (for
+    // grouping I assume?)
+    AggrRole = Qt::UserRole + 2,
+
+    // data(ContentsRole) contains mod data contents as a QVariantList
+    // containing icon paths
+    ContentsRole = Qt::UserRole + 3,
+
+    NameRole = Qt::UserRole + 4,

```

Comment:
This is actually the gameName role from looking at the rest of the code. Just NameRole is ambiguous I think